File: ejercicios/python/ejercicio006.py
from math import *

A = int(input("Ingrese el valor de A (A > 1):? "))
B = int(input("Ingrese el valor de B (B > 1):? "))
if A > 1 and B > 1:
   print("Divisi—n entera " + str(A) + "/" + str(B) + " = " + str(A//B))
   print("Residuo Divisi—n " + str(A) + "%" + str(B) + " = " + str(A%B))
else: 
   print("Error en el ingreso de los datos " + str(A) + "," + str(B))

File: ejercicios/c++/ejercicio006.cpp
#include<iostream>
using namespace std;

int main() {

	int a;
	int b;

	cout<<"Ingrese el valor de A (A > 1):? ";
	cin>>a;
	cout<<"Ingrese el valor de B (B > 1):? ";
	cin>>b;
	if (a>1 && b>1) {
		cout<<"Divisi—n entera "<<a<<"/"<<b<<" = "<<(a/b)<<endl;
		cout<<"Residuo Divisi—n "<<a<<"%"<<b<<" = "<<a%b<<endl;
	} else {
		cout<<"Error en el ingreso de los datos "<<a<<","<<b<<endl;
	}

	return 0;
}

